build/.nativeignore

```diff
@@ -83,6 +86,7 @@ node-pty/binding.gyp
 node-pty/build/**
 node-pty/src/**
 node-pty/tools/**
+node-pty/deps/**
```

Comment:
@Tyriar Does this make sense? `winpty` seems to be leaking here.

Reply:
Makes sense, it's only used during the install step. Thanks for finding :+1: